machines/16x8/Makefile.machine.include

```diff
@@ -1,12 +1,20 @@
-BSG_MACHINE_GLOBAL_X                  = 9
-BSG_MACHINE_GLOBAL_Y                  = 16
+BSG_MACHINE_GLOBAL_X                  = 16
+BSG_MACHINE_GLOBAL_Y                  = 9
+
 BSG_MACHINE_VCACHE_SET                = 128
 BSG_MACHINE_VCACHE_WAY                = 8
 BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS   = 8
+BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 64
```

Comment:
Is BSG_MACHINE_VCACHE_DMA_DATA_WIDTH documented somewhere so I can understand what it is?

Reply:
(This is not really a comment on the PR, per say, but I'd like to know what it's for in general)

Reply:
added a comment in README

Reply:
Thanks!